modules/cij/runner.py

```diff
@@ -576,10 +591,10 @@ def tsuite_setup(trun, declr, enum) -> TestSuite:
     tcase_fpaths = []                               # Load testcase fpaths
     if os.path.exists(suite.fpath):              # From suite-file
         suite_lines = (
-            l.strip() for l in open(suite.fpath).read().splitlines()
+            line.strip() for line in open(suite.fpath).read().splitlines()
```

Comment:
The opened file is never closed - consider doing something like

```
suite_content = ""
with open(suite.fpath) as suitef:
  suite_content = suitef.read()
  
suite_lines = ( .... )
  
```

Reply:
Good point, thanks